### ISSUE TITLE
New model not listed in available models

### ISSUE THREAD

**Issue by huths0lo** [FrSkyRC/ETHOS-Feedback-Community#1845]
I added a new heli to my radio.  The model works fine, up until I switch to a different model.  After that, I have no way to select the previous model, as its not listed amongst the other models.  The bin file is in the same folder as the other models.  I've added another model, and have the same issue with it.

I believe the issue is that when I named the model, I had a space in the name.  This in turn added a space to the actual filename.  Even though I removed the space on the file name, it still doesnt show up.  I recreated the model again; this time with a single word (no space), and it is listed.

Ethos 1.3.0

**Comment by Hartinger** [comment]
Most likely the same problem already reported in #1825.

**Comment by Nicholas-Luoyi** [comment]
Would you please share your model file which cannot be listed here. I created a model with spaces. And it display well.

**Comment by Hartinger** [comment]
I guess it was due to the number of characters, which was (by the deletion of the space) reduced to the maximum that still works. I can also confirm that spaces per se are no problem.

**Comment by huths0lo** [comment]
I had deleted those files, and I'm now wishing I hadnt.  I just created another.  This one has spaces, and is the max character count.  It wont show up in the list as is.  However, I was able to rename with no spaces, and then it showed up.  The files I was having issues with, when I opened this issue had been renamed, and still wouldnt show up.  I wish I could upload them.  But either way, the attached, as is, wont show on my list of models.

[sab goblin fire.zip](https://github.com/FrSkyRC/ETHOS-Feedback-Community/files/9308376/sab.goblin.fire.zip)
